widget/playground/src/containers/SupportedBlockchains/SupportedBlockchains.tsx

```diff
@@ -0,0 +1,64 @@
+import type { Type } from '../../types';
+
+import { ChainsIcon } from '@rango-dev/ui';
+import React from 'react';
+
+import { MultiSelect } from '../../components/MultiSelect/MultiSelect';
+import { getCategoryNetworks } from '../../helpers';
+import { useConfigStore } from '../../store/config';
+import { useMetaStore } from '../../store/meta';
+
+export function SupportedBlockchains({ type }: { type: Type }) {
+  const {
+    config: { from, to },
+    onChangeBlockChains,
+    onChangeBlockChain,
+    onChangeToken,
+  } = useConfigStore();
+  const {
+    meta: { blockchains },
+  } = useMetaStore();
+
+  const selectedType = type === 'Source' ? from : to;
+
+  const configBlockchains = selectedType?.blockchains;
+
+  const allBlockchain = blockchains.map((chain) => {
+    const { shortName: title, logo, name } = chain;
+    return {
+      title,
+      logo,
+      type: name,
+      networks: getCategoryNetworks([chain]),
+    };
+  });
+
+  const handleBlockchainChange = (items?: string[]) => {
+    // Reset default blockchain and token when the default is not among the selected items.
+    if (
```

Comment:
Is it possible to check the value passed to embedded is correct in this situation? (and make sure there is no bug related to playground)

![Recording 2023-10-22 at 12 31 02](https://github.com/rango-exchange/rango-client/assets/99180934/b430ef2b-b21e-431c-a88a-6b36d76146ba)

Reply:
fixed. it was not related to the playground.